modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/Invites/index.js

```diff
@@ -260,7 +274,7 @@ const Invites = () => {
 
 						<a
 							className="font-weight-bold text-neutral-9"
-							href="https://liferay.com/pt"
+							href="https://help.liferay.com/hc/en-us/articles/360018414031-Account-Support"
```

Comment:
This could be a constant or property from remote app, because it could be changed, what you think? @iagochaves , @jairmedeiros 

```
<my-remote-app support-link="https://help.liferay.com/hc/en-us/articles/360018414031-Account-Support">
```

Reply:
Sure, I'll be adding that!